lib/src/rules/no_magic_number_rule.dart

```diff
@@ -12,11 +12,11 @@ class NoMagicNumberRule extends BaseRule {
   static const _warningMessage =
       'Avoid using magic numbers. Extract them to named constants';
 
-  // TODO(shtepin): allow configuration of allowed values
-  static const allowedMagicNumbers = [-1, 0, 1];
+  final List<num> _allowedMagicNumbers;
```

Comment:
@Anrock can you change List on Iterable?

Reply:
@dkrutskikh sure, but what is the point? It's private anyway. For immutability?

Reply:
@Anrock exactly, for immutability.